app/models/playlist.js

```diff
@@ -73,6 +73,16 @@ const Playlist = Backbone.Model.extend({
         if (tracks.type == 'playlist') this.save();
     },
 
+    // Insert a track at a given pos
+    reorderTrack(track, pos) {
+        let tracks = this.get('tracks');
+        let old = tracks.get(track.get('_id'));
```

Comment:
You don't need to get old, coz `old === track`. Just remove `track` from `tracks` collection and add it with `at` param should be sufficient.